mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -37,9 +37,11 @@
 def test_brainvision_data_filters():
     """Test reading raw Brain Vision files
     """
-    raw = _test_raw_reader(read_raw_brainvision,
-                           vhdr_fname=vhdr_highpass_path, montage=montage,
-                           eog=eog)
+    with warnings.catch_warnings(record=True) as w:  # event parsing
+        raw = _test_raw_reader(
+            read_raw_brainvision, vhdr_fname=vhdr_highpass_path,
+            montage=montage, eog=eog)
+    assert_true(all('parse triggers that' in str(ww.message) for ww in w))
```

Comment:
@jona-sassenhagen during the BrainVision event PR #3162, it looks like we made it so that you will pretty much always get a warning when reading BrainVision files like:
```
/home/larsoner/custombuilds/mne-python/mne/io/tests/test_raw.py:37: RuntimeWarning: Currently, 1 trigger(s) will be dropped, such as [Sync On]. Consider using ``event_id`` to parse triggers that do not follow the 'SXXX' pattern.
```
Is `Sync On` always there? If so, perhaps we should omit it from the list of things to warn about, or not warn if it's the only item in the list WDYT?